src/pages/registration/SupportModalContent.tsx

```diff
@@ -1,52 +1,40 @@
-import { Trans, useTranslation } from 'react-i18next';
-import { Link, Typography } from '@mui/material';
-import { Link as RouterLink } from 'react-router-dom';
-import { MessageForm } from '../../components/MessageForm';
-import { addTicketMessage, createTicket } from '../../api/registrationApi';
-import { useDispatch } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+import { fetchRegistrationTickets } from '../../api/registrationApi';
+import { useDispatch, useSelector } from 'react-redux';
 import { setNotification } from '../../redux/notificationSlice';
-import { isErrorStatus, isSuccessStatus } from '../../utils/constants';
-import { UrlPathTemplate } from '../../utils/urlPaths';
+import { useQuery } from '@tanstack/react-query';
+import { RootState } from '../../redux/store';
+import { userCanEditRegistration } from '../../utils/registration-helpers';
+import { Box } from '@mui/material';
+import { ActionPanelContent } from '../public_registration/ActionPanelContent';
+import { Registration } from '../../types/registration.types';
 
 interface SupportModalContentProps {
   closeModal: () => void;
-  registrationId: string;
+  registration: Registration;
 }
 
-export const SupportModalContent = ({ closeModal, registrationId }: SupportModalContentProps) => {
+export const SupportModalContent = ({ registration }: SupportModalContentProps) => {
   const dispatch = useDispatch();
   const { t } = useTranslation();
+  const { user } = useSelector((store: RootState) => store);
 
-  const sendMessage = async (message: string) => {
-    if (message) {
-      // Create ticket
-      const createTicketResponse = await createTicket(registrationId, 'GeneralSupportCase', true);
-      if (isErrorStatus(createTicketResponse.status)) {
-        dispatch(setNotification({ message: t('feedback.error.send_message'), variant: 'error' }));
-      } else if (isSuccessStatus(createTicketResponse.status)) {
-        const ticketId = createTicketResponse.data?.id;
-        if (ticketId) {
-          const addMessageResponse = await addTicketMessage(ticketId, message);
-          if (isErrorStatus(addMessageResponse.status)) {
-            dispatch(setNotification({ message: t('feedback.error.send_message'), variant: 'error' }));
-          } else if (isSuccessStatus(addMessageResponse.status)) {
-            dispatch(setNotification({ message: t('feedback.success.send_message'), variant: 'success' }));
-            closeModal();
-          }
-        }
-      }
-    }
-  };
+  const isRegistrationAdmin = !!registration && userCanEditRegistration(user, registration);
```

Comment:
litt njæ denne da, siden man uansett ikke kan se wizard om man ikke kan endre registrering. Så kan godt fjernes for min del, men det er jo ikke feil at den ligger her heller. Blir bare en ekstra sjekk